# <mayhem_program::instructions::admin::InitializeGlobalParams as anchor_lang::Accounts<mayhem_program::instructions::admin::InitializeGlobalParamsBumps>>::try_accounts::{{closure}}  (yours 4680 B at .text+0x269d0; original 4680 B at .text+0x30cf8)
# distance 1 instructions; evidence: same-callees 15/15; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -58,7 +58,7 @@
 stxdw [r10-272], r1
 stw [r10-128], 4101
 stb [r10-200], 2
-stw [r10-256], 9
+stw [r10-256], 66
 stdw [r10-264], 49
 stdw [r10-280], 0
 mov64 r1, r10
